### ISSUE TITLE
login fails since upgrade from 9.0.2 to 9.1.4 

### ISSUE THREAD

**Issue by leobase** [owncloud/client#5714]
### Expected behaviour
Login to ownCloud site in the client is done in background - connection is established without the need of filling in credentials 

### Actual behaviour
After booting the PC I will be asked for credentials in a loop. Sometimes it works if I wait for several minutes after reboot of the PC and then I need to hit "ok" only once.

### Steps to reproduce
1. Upgrade from 9.0.2 to 9.1.4
2. Reboot Windows Desktop PC
3. ownCloud Desktop Client starts with asking for credentials
4. filling in credentials
5. Hit ok
6. ownCloud Client asks again for the credentials
7. Hitting OK with filled in credentials for 30 times and more, no connection
8. Remove Account from Windows Client
9. Configure the same account again
10. At the end of configuration hitting "Connect" - Client is connected.

### Server configuration
Operating system: Debian 8 Jessie
Web server: Apache 2.4.10-10+deb8u7
Database: mySQL 5.5.54-0+deb8u1
PHP version: 5.6.30+dfsg-0+deb8u1
ownCloud version (see ownCloud admin page): 9.1.4 stable
Updated from an older ownCloud or fresh install: Updated from 9.0.2 and before 8.x
Special configurations (external storage, external authentication, reverse proxy, server-side-encryption): LDAP auth (OpenLDAP).

### Client configuration
Client version: Tested with 2.2.3 / 2.2.4 / 2.3.0 / 2.3.1
Client operating system: Windows 7 x64
OS language: German

Installation path of client: C:\Program Files (x86)\ownCloud

### ownCloud log (data/owncloud.log)
If I hit ok and the credentials aren´t accepted no log is written in the owncloud.log - a few seconds later I get the following errors:
```
{"reqId":"WNk7EgoAAgwAAEchB8oAAAAG","remoteAddr":"87.78.93.65","app":"user_ldap","message":"Configuration` Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.","level":2,"time":"2017-03-27T18:17:22+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7EgoAAgwAAEchB8oAAAAG","remoteAddr":"87.78.93.65","app":"core","message":"Login failed: 'ben' (Remote IP: '87.78.93.65')","level":2,"time":"2017-03-27T18:17:22+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7EgoAAgwAAEchB8oAAAAG","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:17:22+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7HgoAAgwAAAu8xAoAAAAC","remoteAddr":"87.78.93.65","app":"user_ldap","message":"Configuration Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.","level":2,"time":"2017-03-27T18:17:34+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7HgoAAgwAAAu8xAoAAAAC","remoteAddr":"87.78.93.65","app":"core","message":"Login failed: 'ben' (Remote IP: '87.78.93.65')","level":2,"time":"2017-03-27T18:17:34+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7HgoAAgwAAAu8xAoAAAAC","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:17:34+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7QAoAAgwAAAu@jKsAAAAE","remoteAddr":"87.78.93.65","app":"user_ldap","message":"Configuration Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.","level":2,"time":"2017-03-27T18:18:08+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7QAoAAgwAAAu@jKsAAAAE","remoteAddr":"87.78.93.65","app":"core","message":"Login failed: 'ben' (Remote IP: '87.78.93.65')","level":2,"time":"2017-03-27T18:18:08+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7QAoAAgwAAAu@jKsAAAAE","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:18:08+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XgoAAgwAAAu9Lx8AAAAB","remoteAddr":"87.78.93.65","app":"user_ldap","message":"Configuration Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.","level":2,"time":"2017-03-27T18:18:38+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XgoAAgwAAAu9Lx8AAAAB","remoteAddr":"87.78.93.65","app":"core","message":"Login failed: 'ben' (Remote IP: '87.78.93.65')","level":2,"time":"2017-03-27T18:18:38+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XgoAAgwAAAu9Lx8AAAAB","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:18:38+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XwoAAgwAADPppGoAAAAF","remoteAddr":"87.78.93.65","app":"user_ldap","message":"Configuration Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.","level":2,"time":"2017-03-27T18:18:39+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XwoAAgwAADPppGoAAAAF","remoteAddr":"87.78.93.65","app":"core","message":"Login failed: 'ben' (Remote IP: '87.78.93.65')","level":2,"time":"2017-03-27T18:18:39+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
{"reqId":"WNk7XwoAAgwAADPppGoAAAAF","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:18:39+02:00","method":"PROPFIND","url":"\/remote.php\/webdav\/","user":"--"}
```

If I delete the Account in the client and configure it again, connections works and I get this error, which is normal and can´t be solved (I tried every solution in the knowledgebase):
```
{"reqId":"WNk8NwoAAgwAAEchB9AAAAAG","remoteAddr":"87.78.93.65","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 401 No 'Authorization: Basic' header found. Either the client didn't send one, or the server is mis-configured\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\NotAuthenticated\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\Auth\\\\Plugin->beforeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(446): Sabre\\\\Event\\\\EventEmitter->emit('beforeMethod', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(57): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#6 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Auth\\\/Plugin.php\",\"Line\":188,\"User\":false}","level":0,"time":"2017-03-27T18:22:15+02:00","method":"GET","url":"\/remote.php\/webdav\/","user":"--"}
```

Please paste possible errors in the following code block, see https://central.owncloud.org/t/how-to-find-webserver-or-oc-logfile-enable-php-logfile/808 for more info

No errors in Apache error.log
No errors in php.log

### Additional Informations
1. Authentication in Webinterface works perfectly
2. Authentication with WinSCP and WebDav works perfectly
3. Authentication with Android WebDav App works perfectly
4. Issue occurs on 3 different clients (2x windows 7 x64 / 1x Windows Vista x86) 
5. Issue happens on a fresh installed windows client with no ownCloud-Client installed before.
6. Deleted the informations stored at the "Credential manager from Windows" for testing but issue occurs again.
7. Basically my OC installation was done by Debian Packet-Manager (apt-get)

If you will need anything else, don´t hesitate to ask.

Best regards,
Ben

**Comment by guruz** [comment]
Linking https://central.owncloud.org/t/windows-client-login-fails-since-upgrade-from-9-0-2-to-9-1-4/6673/11

**Comment by guruz** [comment]
Do you have the possibility to check with another computer (friend, colleague etc?)?

**Comment by leobase** [comment]
Yes, I´ve done it:
-> See point 4 at "Additional Informations":

Issue occurs on 3 different clients (2x windows 7 x64 / 1x Windows Vista x86)

And point 5:
Issue happens on a fresh installed windows client with no ownCloud-Client installed before.

**Comment by leobase** [comment]
@guruz - If you would test it by yourself, I can give you user access to the OC installation

**Comment by ckamm** [comment]
@leobase I'd be up for testing connectivity against your server. Would you email the credentials for a testing account to mail at ckamm de?

**Comment by ckamm** [comment]
From your log it looks like it might a server/ldap configuration issue though?

**Comment by guruz** [comment]
Yep, please feel free to send credentials to @ckamm

**Comment by leobase** [comment]
Thanks for help, I´ll send credential to the given address. 

If it would be a LDAP issue, why will it work if I delete and add the user-acc in the Windows client? And why will it work with WebDav and Web-Portal?

**Comment by leobase** [comment]
Mail was written to the provided address. 

Thanks for your support.

**Comment by ckamm** [comment]
@leobase Thank you. I can indeed reproduce the issue even on linux. I'll not get around to digging into it before Monday though.

Btw, your email got marked as spam by google :/

**Comment by guruz** [comment]
@leobase Are you using PHP-FPM?

**Comment by leobase** [comment]
@guruz I´ve installed PHP-FPM (php5-fpm 5.6.30+dfsg-0+deb8u1) but it is not configured in the virtualhost for the OC site. For testing purposes I disabled the service and tested the login again, but same behaviour.

**Comment by leobase** [comment]
@ckamm Thanks for your time :-)

**Comment by ckamm** [comment]
@ogoffart @PVince81 @leobase I've been staring at this for a while and think this must be a server issue.

Details on what happens when it fails (account is already configured when client starts):
```
GET status.php
no cookies, with bad basic auth (no password)
RESPONSE
Set-Cookie:                     ocwoue...=u05lkk...; path=/; HttpOnly                     
Set-Cookie:                     oc_sessionPassphrase=YkMnI...; path=/; secure; httponly                                                   

GET status.php
with good basic auth, cookies from above
RESPONSE
no new cookies

PROPFIND
with good basic auth, cookies from above
RESPONSE
Set-Cookie:                     ocwoue...=243jgknd...; path=/; secure; HttpOnly

GET capabilities.php
with good basic auth, cookies updated
RESPONSE
Auth failed
```
However, this structure of this flow is identical when I try against demo.owncloud.org, except that the final session cookie works and capabilities are retrieved.

For completeness, here's the "Add new account" flow, that works correctly:
```
PROPFIND
no cookies, good basic auth
RESPONSE
Set-Cookie:                     ocwoue...=e1o1g2j...; path=/; HttpOnly                     
Set-Cookie:                     oc_sessionPassphrase=HHsPgOCrqV...; secure; httponly                                                      
Set-Cookie:                     ocwoue...=vb1443...; path=/; secure; HttpOnly             

PROPFIND
above, using the session cookie marked as secure
RESPONSE
Set-Cookie:                     ocwoue...=a3n1k2...; path=/; secure; HttpOnly             
Set-Cookie:                     ocwoue...=gm2o462...; path=/; secure; HttpOnly             

Uses the `gm2` cookie from then on and works, can GET capabilities.
```

**Comment by leobase** [comment]
@ckamm Thanks for analyzing so deeply - is there something I can do?

**Comment by ckamm** [comment]
@leobase I don't know, unfortunately.

The errors in your owncloud.log in response to the PROPFIND with the correct auth data may be most indicative:
```
Configuration` Error (prefix ): either no password is given for theuser agent or a password is given, but not anLDAP agent.
Login failed: 'ben' (Remote IP: ...)
Exception: {\"Message\":\"HTTP\\\/1.1 401 Username or password was incorrect\
```

**Comment by leobase** [comment]
Just for information - I have updated the Windows Client to the latest version yesterday, but the error still occurs.

**Comment by guruz** [comment]
@leobase It's either a server setup or a server issue.

@PVince81 do you have an idea?

**Comment by leobase** [comment]
What I noticed in the last days, that in the new client version there seems to be another "time-out" value or something like this. If you will not hit "ok" in the owncloud-client-window when it asks for credentials after a reboot of the PC, and if you will wait round about 5 minutes and hit "ok", then                                                                     the window closes, and in the tray-bar you can see the ownCloud icon with a red "x". If you will force a sync then, then it will connect and work as expected. This was possible in the old client version too but you had to wait a lot more time, and it only worked at 50% of the connection attempts. With the new client version it works more often, I would say in 75% of the connection attempts.

Maybe this helps figuring out what is going wrong :-)

**Comment by ckamm** [comment]
@leobase Is this timeout issue a separate issue or related to this ticket? You don't have to wait for 5 minutes to trigger the issue you described here, right? I'll make a new ticket about timeouts during the credentials and ssl dialogs.

**Comment by leobase** [comment]
Hi, from my point of view it is related to this case, because, the client is asking for the credentials after a reboot from the PC. If you enter the credentials and hit "ok", the window pops up again and you can´t connect to the site. If you do so, you can try to connect for hours and it won´t work. If you will wait for 5 minutes after the reboot, let the ownClound client winodw stay open and do nothing with it and then hit ok, the window disappears and the ownCloud-Symbol can be found in the tray-bar. And if you then force the sync, or wait another few minutes, then it will sync. Therefore, the problem I described in this case happens only directly after starting the ownCloud-Client, if some "time-out" value has gone, then the client seems to accept the credentials.

**Comment by ckamm** [comment]
@leobase I see, thanks for clarifying. So you can use the timeout as a workaround. Forcing a sync and account creation are similar in that they don't fetch status.php first. Still likely to be a server or setup issue.

**Comment by leobase** [comment]
This could be, yes, I just wanted to give you all informations I found related to this case.

**Comment by leobase** [comment]
Sorry for asking, but as it is removed now from next milestone, means, I need to wait longer for a fix?

**Comment by ckamm** [comment]
@leobase The complication is that this is not a client issue: the login process works for everyone else, we can't fix it because it does what it should. Similarly, it's unlikely to be a server problem because then we should have seen more reports about this kind of issue, but as far as I'm aware we haven't.

The highest likelihood is on a unique server configuration or setup issue on your end which we haven't figured out yet. If we knew what was going wrong in client or server we'd create a fix, but currently I'm not convinced something is wrong in the first place.

**Comment by ogoffart** [comment]
Please ask on https://central.owncloud.org/ (See https://owncloud.org/support )
The bug tracker is not a support channel.

**Comment by leobase** [comment]
@ogoffart Yes, this was my first step, then I was intended to open a bug here, because no one could help me..... 

https://central.owncloud.org/t/windows-client-login-fails-since-upgrade-from-9-0-2-to-9-1-4/6673

And now, should I open a new question at central?

**Comment by leobase** [comment]
@ogoffart - As I´m reading this again, where have I used this as a support channel? I only gave informations about a problem, where someone else is/was working on. This is an affront to me, I had only done what others said is the best way to get a solution!

It is simple to just close problems you don´t like to invest time - I´m working as a technical support engineer for years, and your behaviour is not professional. It seems that ownCloud really don´t want a community. I should switch to nextcloud, thanks for nothing!

**Comment by guruz** [comment]
@leobase I think it's hard to help if an issue happens only for one user (out of hundreds of thousand users or more) 
:-/

That's why the preliminary conclusion is that it's a configuration/setup issue.

Sorry.

**Comment by ogoffart** [comment]
@leobase  I apologize if I sounded rude.  I just meant to close the issue on this bug tracker since it is not a problem on the client.  You can continue the discussion on central, or open a new question. Maybe by setting different tag you have better chances.
Please don't take the closing of this bug as an insult. There is no reason to keep this open on the client bug tracker because there is no evidence of a bug in the client.
We still welcome bug reports from the community of course. (Thanks for reporting bugs).

(And even if you switch to nextcloud, we'd still look at your bug reports that you report here, for the client)

**Comment by leobase** [comment]
Thanks, yes it sounded a bit of rude.

If it is no bug in the client, yes, close this issue. But as it worked before the update to 9.1.4 and as it works in some scenarios now, and as I changed nothing then the update - it seems like a bug.

**Comment by guruz** [comment]
@leobase Try to find other users here or in core repo or on central that have this issue.
Try to check more logs (error.log?)
Try a clean install directory of oC with existing DB?
Try to switch between Apache and nginx etc
Just to get more data

**Comment by kdslkdsaldsal** [comment]
Cross-Ref: https://github.com/owncloud/core/issues/28245
